lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -289,12 +292,19 @@ def add_to_list_top
             self[position_column] = acts_as_list_top
```

Comment:
For completeness sake, let's set `@scope_changed = false` here too since it's possible this gets run instead of `_bottom`. I know it has no effect but it may prevent a problem in the future.